src/commands/selectSignImage.ts

```diff
@@ -0,0 +1,102 @@
+import * as vscode from 'vscode';
+import { logInToDockerCli } from './registries/logInToDockerCli';
+import { checkCLI } from '../utils/checkCLI';
+import { execAsync } from '../utils/execAsync';
+
+const cliName = 'notation';
+const goToNotationButton = 'Download Notation';
+const NotationURL = 'https://notaryproject.dev/docs/installation/cli/';
+const errorMessage = `Notation not found: `;
+
+//transforms the key list string into an array of objects
+async function sortOutput(Key_String_Values: string ){
+    var dataArray = new Array();
+    var keyChoices = Key_String_Values.split("\n");
+    for (let i = 1; i < keyChoices.length; i++) { 
+      const values = keyChoices[i].trim().split(/ {2,} /); 
+      var itemObject; 
+        for (let j = 0; j < 1; j++) { 
+            const value = values[j];
+            if (value.indexOf('*')> -1){
+                const keyWithoutAsterisk = value.replace('* ', '');
+                itemObject = {label: keyWithoutAsterisk, description: "Current default"};
+            } else {
+                itemObject = {label: value};
+            }
+        }
+      dataArray.push(itemObject);
+    }
+    return dataArray;
+}
+
+//adds precreated azure key to notation
+async function addNewKey() { 
+    var key_Name = await vscode.window.showInputBox({prompt: "Input the key name"});
+    var akv_Name = await vscode.window.showInputBox({prompt: "Input the azure key vault name"});
```

Comment:
are you sure notation command takes in the azure key vault name? Shouldn't it be the AKV key ID instead?